spec/features/va_facility_spec.rb

```diff
@@ -227,6 +227,15 @@ def check_search_results_as_guest_user(container_selector)
       expect(page).to have_current_path(va_facility_path(@facility_1))
     end
 
+    it 'should render a gmap' do
+      login_and_visit_facility_page
+      find(:css, '.va_facilities')
+      visit '/facilities/a-first-facility-test-common-name'
+      sleep 0.5
+
+      expect(page).to have_selector('#va_facility_map', visible: true)
```

Comment:
This ID will still render even if the map JS doesn't load. How about `.gm-styles` instead?

Reply:
good catch!